src/igvfd/tests/test_schema_treatment.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+
+def test_treatment_amount_dependency(testapp):
+    item = {
+        'treatment_term_name': 'lactate',
+        'treatment_term_id': 'CHEBI:24996',
+        'treatment_type': 'chemical',
+        'amount': '10'
+    }
+    return testapp.post_json('/treatment', item, expect_errors=True)
```

Comment:
This return statement means that lines 12-19 never get executed.  You will need to change the line to res = like in line 23.  However, this test is of a dependency that is redundant.  You do not need to ensure that amount units is required if amount is there, it is always required.  So, if amount and amount_units are required, this test from lines 4-19 is not required.